### PR TITLE
Tweaked Select component to match latest specs

### PR DESCRIPTION
**Pull request by pppemo**

### CODE REVIEW

File: packages/react/src/components/Select/index.js [hunk 1]
```diff
@@ -36,7 +36,11 @@ const Select = ({
         ))}
       </select>
 
-      <Icon icon="IconTopDown" className="FormEl-icon" size={ICON_SIZES.S} />
+      <Icon
+        icon="IconChevronDown"
```
Comment (ugogo) [inline packages/react/src/components/Select/index.js]:
```suggestion
        icon={Icon.IconChevronDown}
```

File: packages/react/src/components/Select/index.stories.js [hunk 2]
```diff
@@ -34,9 +39,13 @@ storiesOf("Form controls/Select", module)
       </div>
     ))
   )
-  .add("With custom props", () => (
+  .add("Playground", () => (
     <>
```
Comment (ugogo) [inline packages/react/src/components/Select/index.stories.js]: This `Fragment` is useless 😉

File: packages/react/src/components/Select/index.stories.js [hunk 3]
```diff
@@ -34,9 +39,13 @@ storiesOf("Form controls/Select", module)
...
-      <Select {...defaultProps()} />
-      <Select {...defaultProps()} fill />
+      <Select
+        options={mapStringsToSelectOptions(
```
Comment (ugogo) [inline packages/react/src/components/Select/index.stories.js]: 👌

File: packages/react/src/components/Select/index.js (hunk 1 above)
Comment (pppemo) [reply packages/react/src/components/Select/index.js]: 👍

File: packages/react/src/components/Select/index.stories.js (hunk 2 above)
Comment (pppemo) [reply packages/react/src/components/Select/index.stories.js]: 👍 Leftover

**Comment by ugogo** [comment]
Merged into #63

**Pull request closed without merging**